contracts/DefaultStrategy.sol

```diff
@@ -568,17 +601,17 @@ contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Ma
         address votedGroup = activeGroups.getHead();
         uint256 groupsIndex;
 
-        for (groupsIndex = 0; groupsIndex < maxGroupCount; groupsIndex++) {
-            if (votesAmount == 0 || votedGroup == address(0)) {
+        while (groupsIndex < maxGroupCount) {
```

Comment:
Why the change from `for` to `while`?

Reply:
It felt to me as more readable - I could switch it back if you dont think that's the case

Reply:
I did refactor it a bit more